cfpb/cfgov-refresh
# CEIA: Final megamenu & URL updates

Make final updates to megamenu and URLs for CEIA project

## Changes

- Update text and order of some megamenu links in `Practitioner Resources` vertical
- Move remaining pages under `practitioner-resources`
- Add redirects for moved pages

## Testing

1. Check out branch
2. Verify megamenu changes:
    - `Practitioner Resources` section of menu should match below screenshot
    - `Economically vulnerable consumers`, `Servicemembers & Veterans`, and `Students & Student Loan Borrowers` links should go to appropriate pages under `/practitioner-resources/`
3. Verify that the existing URLs are redirected to the new page locations under `/practitioner-resources/`:
    - http://localhost:8000/empowerment/
    - http://localhost:8000/servicemembers/
    - http://localhost:8000/servicemembers/on-demand-forums-and-tools/
    - http://localhost:8000/servicemembers/additionalresources/
    - http://localhost:8000/servicemembers/planning/
    - http://localhost:8000/servicemembers/planning/creativesavingsstrategies/
    - http://localhost:8000/students/
    - http://localhost:8000/students/knowbeforeyouowe/
    - http://localhost:8000/students/helping-borrowers-find-ways-to-stay-afloat/

## Screenshots

### Updated megamenu
<img width="1194" alt="screen shot 2017-10-30 at 12 46 38 pm" src="https://user-images.githubusercontent.com/778171/32283474-a58a90f6-bee1-11e7-888e-bf4a22bc25ef.png">

## Checklist
